ee-9/feature-pack/pom.xml

```diff
@@ -440,7 +440,31 @@
             <scope>provided</scope>
         </dependency>
 
-        <!-- module and copy artifact dependencies -->
+	<!-- module and copy artifact dependencies -->
```

Comment:
whitespace :)

Please check that in general in this file.

Reply:
I updated my vim rules to show tab characters so that I can be more mindful of this occurring.  Sorry about the repeat offense.